config/v1/types_infrastructure.go

```diff
@@ -478,6 +478,11 @@ type AWSPlatformStatus struct {
 	// +kubebuilder:validation:MaxItems=25
 	// +optional
 	ResourceTags []AWSResourceTag `json:"resourceTags,omitempty"`
+
+	// dnsConfig contains information about API and API-Int Load Balancer
+	// and DNS configuration.
+	// +optional
+	DNSConfig DNSAndLBConfiguration `json:"dnsConfig,omitempty"`
```

Comment:
Is this a tech preview feature? If so, we need to add the appropriate tag to make it only show up on tech preview/custom feature sets